### PR TITLE
chore(explorer): migrate pending PRs from old repo

### PR DESCRIPTION
**Pull request by shoom3301**
# Summary

Just copy-pasted valuable changes from pending PRs:
https://github.com/cowprotocol/explorer/pull/698
https://github.com/cowprotocol/explorer/pull/672

### CODE REVIEW

**Review by alfetopito — APPROVED** [review]
Working great

File: apps/explorer/.env.example [hunk 1]
```diff
@@ -0,0 +1,16 @@
+README.md# Enables mock mode (default = true)
```
Comment (anxolin) [inline apps/explorer/.env.example:1]: this is not supoused to be here, right?

File: apps/explorer/.env.example [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+MOCK=true
+
+# Enables autoconnect for mock mode (default = true)
+AUTOCONNECT=true
+
+# Public IPFS gateway
+REACT_APP_IPFS_READ_URI=https://cloudflare-ipfs.com/ipfs
```
Comment (anxolin) [inline apps/explorer/.env.example:8]: probably we don't need the REACT_APP now?

File: apps/explorer/README.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+# Explorer
```
Comment (anxolin) [inline apps/explorer/README.md:1]: should this file be liked from the main README.md?

File: apps/explorer/.env.example (hunk 2 above)
Comment (anxolin) [reply apps/explorer/.env.example:8]: Did you test this work?

File: apps/explorer/.env.example (hunk 1 above)
Comment (shoom3301) [reply apps/explorer/.env.example:1]: Oooops, thanks!

File: apps/explorer/.env.example (hunk 2 above)
Comment (shoom3301) [reply apps/explorer/.env.example:8]: Yes, it still in use for app-data decoding

File: apps/explorer/README.md (hunk 3 above)
Comment (shoom3301) [reply apps/explorer/README.md:1]: Makes sense! Added it

**Pull request merged**